coupling_components/solver_wrappers/openfoam/openfoam.py

```diff
@@ -564,14 +572,17 @@ def read_modify_controldict(self):
         control_dict = re.sub(r'endTime' + of_io.delimiter + of_io.float_pattern, f'{"endTime":<16}1e15', control_dict)
 
         # delete previously defined coconut functions
-        coconut_start_string = '\n// CoCoNuT function objects'
+        coconut_start_string = '\n\n// CoCoNuT function objects'
         control_dict = re.sub(coconut_start_string + r'.*', '', control_dict, flags=re.S)
 
         with open(file_name, 'w') as control_dict_file:
             control_dict_file.write(control_dict)
             control_dict_file.write(coconut_start_string + '\n')
 
-            control_dict_file.write('boundaryNames (' + ' '.join(self.boundary_names) + ');\n\n')
+            control_dict_file.write('boundaryNames   (' + ' '.join(self.boundary_names) + ');\n\n')
```

Comment:
Is the additional space after 'boundaryNames' intentional?

Reply:
Yes, aligning in OF is typically done with _tabs_ of 4 spaces; the two additional spaces assure the bracket starts at a _new tab_.